MAINTAINERS.md

```diff
@@ -0,0 +1,143 @@
+<!--
+******************************************************************************
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/-->
+
+# Introduction
+
+This document defines roles in the oneTBB project.
+
+# Roles and responsibilities
+
+oneTBB project defines three main roles:
+ * [Contributor](#contributor)
+ * [Code Owner](#code-Owner)
+ * [Maintainer](#maintainer)
+
+[permissions]: https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permissions-for-each-role
+
+|                                                                                                                                             |       Contributor       |       Code Owner        |       Maintainer        |
+| :------------------------------------------------------------------------------------------------------------------------------------------ | :---------------------: | :---------------------: | :---------------------: |
+| _Responsibilities_                                                                                                                          |                         |                         |                         |
+| Follow the Code of Conduct                                                                                                                  |            ✓            |            ✓           |            ✓            |
+| Follow Contribution Guidelines                                                                                                              |            ✓            |            ✓           |            ✓            |
+| Enforce Contribution Guidelines                                                                                                             |            ✗            |            ✓           |            ✓            |
```

Comment:
I would prefer to keep it because it is aligned with https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permissions-for-each-role